http2_util/server_debug.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Communication Service/Software Laboratory, National Chiao Tung University (free5gc.org)
+//
+// SPDX-License-Identifier: Apache-2.0
+
+//+build debug
+
```

Comment:
```suggestion
```

Reply:
Those comments affect the build process, if I remove them, both files will be built simultaneously and will conflict.

I can simply remove the debug version if we do not require it and then remove the comments from the `server.go` file.

WDYT?

Reply:
Understood. Then, let's just leave them there. Thanks for the clarification!